legate/tester/args.py

```diff
@@ -187,6 +187,39 @@
 )
 
 
+test_opts.add_argument(
+    "--cov-bin",
+    dest="cov_bin",
+    action="store",
```

Comment:
"store" is the default action, this is not necessary (here and below)

https://docs.python.org/3/library/argparse.html#action

Reply:
Fixed. Thanks.